apps/store/src/services/formTemplate/data/NO_ACCIDENT_UI.ts

```diff
@@ -0,0 +1,32 @@
+import { FormTemplateUISchema } from '../FormTemplate.types'
+
+export const NO_ACCIDENT_UI: FormTemplateUISchema = {
+  layout: {
+    sections: [
+      {
+        id: 'your-info',
+        title: { key: 'Your home' },
```

Comment:
Shouldn't be the following here?
```suggestion
        title: { key: 'Your Info' },
```